plugins/org.eclipse.fordiac.ide.model.search/src/org/eclipse/fordiac/ide/model/search/ModelSearchPattern.java

```diff
@@ -13,21 +13,25 @@
  *******************************************************************************/
 package org.eclipse.fordiac.ide.model.search;
 
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class ModelSearchPattern { // extends SearchPattern
 
-	private final String toTest;
-	private Pattern searchPattern;
-	private final String searchString;
+	private final Pattern searchPattern;
+	private final Pattern preScannSTPattern;
+	private final String preScannSTStringPattern;
 
-	public ModelSearchPattern(final String toTest, final ModelQuerySpec modelQuerySpec) {
-		this.toTest = toTest;
-		this.searchString = modelQuerySpec.getSearchString();
+	public ModelSearchPattern(final ModelQuerySpec modelQuerySpec) {
+		modelQuerySpec.getSearchString();
```

Comment:
This statement seems to have no effect?

Reply:
yes that should have been removed. Done.